test/realtimecontroller/DefaultRealtimeController.test.ts

```diff
@@ -1195,29 +1205,6 @@ describe('DefaultRealtimeController', () => {
       expect(fatal.calledWith(matchUn)).to.be.true;
     });
 
-    it('handles broken volume callbacks', () => {
```

Comment:
Why was this removed?

Reply:
As commented in the PR description, there is nothing in the code that can throw error anymore so this test is no longer valid.
> There is no longer anything that would cause the try/catch to occur, so I removed that code to fix coverage.